src/java/org/apache/cassandra/tools/nodetool/Ring.java

```diff
@@ -177,8 +184,39 @@ else if (movingNodes.contains(endpoint))
 
             String load = loadMap.getOrDefault(endpoint, "?");
             String owns = stat.owns != null && showEffectiveOwnership? new DecimalFormat("##0.00%").format(stat.owns) : "?";
-            out.printf(format, stat.ipOrDns(printPort), rack, status, state, load, owns, stat.token);
+            List<Object> result = Lists.newArrayList(stat.ipOrDns(printPort), rack, status, state, load, owns, stat.token);
+            if (hasCms)
+                result.add(String.valueOf(cmsProxy.isCurrentMember(stat.endpointWithPort)));
```

Comment:
I think instead of calling this for every node in the ring, we could have a `Set<String> cms = cmsProxy.getCurrentCMS()` outside the loop and just check that

Also, would it make sense to output `CMS` on the cms nodes? To make it easier to `nodetool ring --cms | grep CMS` (I don't think we currently output anything else containing true/false, but we could in the future)